tools/workspace-plugin/src/generators/react-library/files/src/testing/isConformant.ts__tmpl__

```diff
@@ -9,6 +9,7 @@ export function isConformant<TProps = {}>(
     tsConfig: { configName: 'tsconfig.spec.json' },
     componentPath: require.main?.filename.replace('.test', ''),
     extraTests: griffelTests as TestObject<TProps>,
+    disabledTests: ['consistent-callback-args'],
```

Comment:
This can be removed because of the changes in #30376, right?

Reply:
Yeah that's correct. I'll remove this